browser/ui/webui/brave_tip_ui.cc

```diff
@@ -470,25 +470,23 @@ void TipMessageHandler::GetShareURLCallback(const std::string& url) {
 void TipMessageHandler::FetchBalanceCallback(
     const ledger::mojom::Result result,
     ledger::mojom::BalancePtr balance) {
-  if (!IsJavascriptAllowed()) {
+  if (!IsJavascriptAllowed() || !balance) {
```

Comment:
Shouldn't `!balance` be considered as if `result == mojom::Result::LEDGER_ERROR`?

Reply:
Good point - after looking more closely this handler looks good as-is.